selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsTest.java

```diff
@@ -124,7 +135,15 @@ public void workingWithEnvVariables() {
 
   @Test
   public void workingWithInstallers() {
-    dashboardWorkspace.selectTabInWorspaceMenu(TabNames.INSTALLERS);
+    dashboardWorkspace.selectTabInWorspaceMenu(INSTALLERS);
+
+    // check both versions of the 'Workspace API' installer
+    Assert.assertTrue(dashboardWorkspace.getInstallerState("Workspace API", "1.0.1"));
+    Assert.assertFalse(dashboardWorkspace.getInstallerState("Workspace API", "1.0.0"));
+    dashboardWorkspace.switchInstallerState("Workspace API", "1.0.1");
```

Comment:
Taking in mind four possible states of Installer State column in the **Installers** Ttab of workspace configuration in Dashboard:
![screenshot from 2017-12-04 15-41-49](https://user-images.githubusercontent.com/1197777/33555683-ebcdcbc2-d909-11e7-96d1-fd4011d4a678.png)

I would suggest to test on four states of the value on UI:
- `boolean isEnabled()`
- `boolean isAccessable()`